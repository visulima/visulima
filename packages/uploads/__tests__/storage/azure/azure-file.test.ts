import { describe, expect, it } from "vitest";

import AzureFile from "../../../src/storage/azure/azure-file";

describe("AzureFile", () => {
    it("should have extra params", () => {
        const file = new AzureFile({ metadata: { foo: "bar" } });

        file.requestId = "requestId";
        file.uri = "uri";
        file.id = "PsRqecfPghVJE6veB-IcX";
        file.originalName = "PsRqecfPghVJE6veB-IcX";

        expect(file).toMatchSnapshot();
    });
});
