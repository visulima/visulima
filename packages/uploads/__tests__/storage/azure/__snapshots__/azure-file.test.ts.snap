// Vitest Snapshot v1

exports[`AzureFile > should have extra params 1`] = `
AzureFile {
  "bytesWritten": NaN,
  "content": undefined,
  "contentType": "application/octet-stream",
  "createdAt": undefined,
  "expiredAt": undefined,
  "hash": undefined,
  "id": "njB3IRUlR63kPk-hhnm2T",
  "metadata": {
    "foo": "bar",
  },
  "modifiedAt": undefined,
  "name": "",
  "originalName": "njB3IRUlR63kPk-hhnm2T",
  "requestId": "requestId",
  "size": undefined,
  "status": undefined,
  "uri": "uri",
}
`;
